src/ZipkinOpenTracing/Span.php

```diff
@@ -155,6 +155,14 @@ public function setTag($key, $value)
      */
     public function log(array $fields = [], $timestamp = null)
     {
+        if ($timestamp === null) {
+            $timestamp = microtime(true);
+        } elseif ($timestamp instanceof \DateTimeInterface) {
+            $timestamp = $timestamp->getTimestamp();
+        }
+
+        $timestamp = (int) ($timestamp * 1000 * 1000);
```

Comment:
Not sure about this. What if someone passes `microtime`? I think we should cast as `DateTime` and then convert to microtime when sending to `zipkin`.

Reply:
That would be alright but additionally we have to multiply as it is done on https://github.com/jcchavezs/zipkin-php/blob/master/src/Zipkin/Timestamp.php#L12

So cases of
* $span->log($fields, microtime(true));
* $span->log($fields, new \DateTime());
* $span->log($fields)

are okay